go/runtime/committee/client.go

```diff
@@ -132,6 +135,24 @@ type clientConnState struct {
 	resolverCleanupCb func()
 }
 
+// DelayedClose closes the connection after the given delay. This method does not block the caller.
+func (cs *clientConnState) DelayedClose(delay time.Duration) {
+	go func() {
+		time.Sleep(delay)
+		cs.Close()
+	}()
+}
+
+// Close closes the connection.
+func (cs *clientConnState) Close() {
```

Comment:
Should this nil out the relevant members?

Reply:
Yeah, fixed.